bundles/specmate-administration/src/com/specmate/administration/api/ESpecmateStatus.java

```diff
@@ -0,0 +1,25 @@
+package com.specmate.administration.api;
+
+public enum ESpecmateStatus {
+
+	NORMAL(ESpecmateStatus.NORMAL_NAME, false), MAINTENANCE("maintenance", true);
+

```

Comment:
Use MAINTENANCE_NAME constant instead of literal

Reply:
done